src/pip/_internal/operations/install/wheel.py

```diff
@@ -432,7 +495,7 @@ def __init__(self, file):
         self.changed = False
 
     def save(self):
-        # type: () -> None
+        # type: (ScriptFile) -> None
```

Comment:
hah, this one is my fault. i had added a `str` argument to this function and forgot to revert the type annotation!

```suggestion
        # type: () -> None
```

Reply:
Ha! I never played with type annotations, so I thought the first argument was the type of self!